examples/tutorials/curation/plot_3_upload_a_model.py

```diff
@@ -0,0 +1,126 @@
+"""
```

Comment:
Not sure about this, but could this be a subsection of the above 'how to train a model' tutorial, as both are reasonably short? It was a little confusing to see three tutorials on the automated curation stuff, though compartmentalization is also very nice.

Reply:
Discussed above